releng/hu.bme.mit.massif.import.parent/pom.xml

```diff
@@ -14,8 +14,8 @@
 
 	</modules>
 	<properties>
-		<tycho.version>0.18.0</tycho.version>
-		<xtend.compiler.version>2.4.2</xtend.compiler.version>
+		<tycho.version>2.0.0</tycho.version>
```

Comment:
This is almost certainly incorrect. The latest tycho release is 1.2.0 (AFAIK).

Reply:
Based on [this list](https://wiki.eclipse.org/Tycho/Release_Notes), the latest release is 1.1.0 - so I'll put that in.

Reply:
Tycho 1.2 is out (http://search.maven.org/#artifactdetails%7Corg.eclipse.tycho%7Ctycho%7C1.2.0%7Cpom and https://wiki.eclipse.org/Tycho/Release_Notes/1.2) - feel free to use it.

Reply:
OK, thank you!